internal/cmd/internal/olmv1/extension_uninstall.go

```diff
@@ -8,26 +8,26 @@ import (
 	"github.com/operator-framework/kubectl-operator/pkg/action"
 )
 
-func NewOperatorUninstallCmd(cfg *action.Configuration) *cobra.Command {
-	u := v1action.NewOperatorUninstall(cfg)
+func NewExtensionUninstallCmd(cfg *action.Configuration) *cobra.Command {
+	u := v1action.NewExtensionUninstall(cfg)
 	u.Logf = log.Printf
 
 	cmd := &cobra.Command{
-		Use:   "uninstall <operator>",
-		Short: "Uninstall an operator",
-		Long: `Uninstall deletes the named Operator object.
+		Use:   "uninstall <extension>",
+		Short: "Uninstall an extension",
+		Long: `Uninstall deletes the named extension object.
 
 Warning: this command permanently deletes objects from the cluster. If the
-uninstalled Operator bundle contains CRDs, the CRDs will be deleted, which
+uninstalled extension bundle contains CRDs, the CRDs will be deleted, which
```

Comment:
we should not use `extension` and `bundle` at the same time. Use of `bundle` is  associated with registry v1 content  but not much with cluster extension.

Reply:
there is no functional change, it's just one-word rename, so I think bundle(s) are still being used to install operators/extensions.

That said, this is a simple rename PR and @LalatenduMohanty is working on install/uninstall commands that this comment touches. How about we have those discussions on that incoming PR where they belong?